extensions/smallrye-jwt/deployment/src/test/java/io/quarkus/jwt/test/SignSecretKeyUnitTest.java

```diff
@@ -0,0 +1,38 @@
+package io.quarkus.jwt.test;
+
+import static org.hamcrest.Matchers.equalTo;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusUnitTest;
+import io.restassured.RestAssured;
+import io.smallrye.jwt.build.Jwt;
+
+public class SignSecretKeyUnitTest {
+    private static Class<?>[] testClasses = {
+            DefaultGroupsEndpoint.class
+    };
```

Comment:
I think this can be moved to [line 20](https://github.com/quarkusio/quarkus/pull/25653/files#diff-93eb02d1d5e2c91539cdb7ee5bd9a90a3d4bc81272e289bcc8d6fad3f7187e07R20)

Reply:
@gastaldi Sure, just updated it